madhav/src/main/java/com/lftechnology/phpjava/strings/NameSplitter.java

```diff
@@ -0,0 +1,37 @@
+package main.java.com.lftechnology.phpjava.strings;
+
+import main.java.com.lftechnology.phpjava.main.Scanner;
+
+/**
+ * 
+ * @author madhav
+ *
+ */
+public class NameSplitter {
+    /**
+     * Gets input from user(in console) assumed to be full name, and displays extracted salutation and name
+     * 
+     * @author madhav
+     */
+    public void getAndExtractname() {
+        String allName = "";
+
+        while (allName == "") {
+            System.out.println("Enter Your Name(with Salutation): ");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)